SocialCareCaseViewerApi/V1/UseCase/WarningNoteUseCase.cs

```diff
@@ -22,12 +20,14 @@ public PostWarningNoteResponse ExecutePost(PostWarningNoteRequest request)
             return _databaseGateway.PostWarningNote(request);
         }
 
-        public List<WarningNote> ExecuteGet(GetWarningNoteRequest request)
+        public ListWarningNotesResponse ExecuteGet(long personId)
```

Comment:
If you agree with SocialCareCareCaseViewerApiController comment this could be reverted.